From 3182e1174fe6bdd3b1fbaf535d63241793c1e982 Mon Sep 17 00:00:00 2001
From: Nobuhiro Iwamatsu <iwamatsu@nigauri.org>
Date: Sun, 5 Jun 2011 15:28:58 +0100
Subject: Update lengthen time-out time for t/re/re.t.

Bug-Debian: http://bugs.debian.org/626125
Bug: http://rt.perl.org/rt3/Public/Bug/Display.html?id=92226
Origin: upstream, http://perl5.git.perl.org/perl.git/commitdiff/7c47ab277d9951f7d3b1f59075afbc754894081c

When we carry out this test on SH4, it becomes the time-out.
2 seconds are set in watchdog, but are too short for SH4.
This patch was changed for 10 seconds.

Patch-Name: fixes/re.t_sh4_timeout.diff
---
 t/re/re.t |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/t/re/re.t b/t/re/re.t
index 249c6dd..84a5d1d 100755
--- a/t/re/re.t
+++ b/t/re/re.t
@@ -54,7 +54,7 @@ if ('1234'=~/(?:(?<A>\d)|(?<C>!))(?<B>\d)(?<A>\d)(?<B>\d)/){
 
     { # Keep this test last, as whole script will be interrupted if times out
         # Bug #72998; this can loop 
-        watchdog(2);
+        watchdog(10);
         eval '"\x{100}\x{FB00}" =~ /\x{100}\N{U+66}+/i';
         pass("Didn't loop");
     }
